docs/source/json-api/index.rst

```diff
@@ -977,8 +977,8 @@ Where
 - ``displayName`` -- optional human readable name associated with the party. Might not be unique,
 - ``isLocal`` -- true if party is hosted by the backing participant.
 
-HTTP Response OK(200) with Unknown Parties Warning
-==================================================
+Nonempty Response with Unknown Parties Warning
```

Comment:
I would merge this section and the one about empty responses and say here that the result might be an empty list if none of the parties is known.
```suggestion
Response with Unknown Parties Warning
```